### DISCUSSION TITLE
How to create github action without using action.yml and publish into marketplace

### DISCUSSION THREAD

**Discussion by ajilraju** [community/community#25551] [Actions]
<p>I have seen the many repo doesn’t contain any action.yml file and it’s only contain Dockerfile with</p>
<pre><code>LABEL "com.github.actions.name"=""
LABEL "com.github.actions.description"=""
LABEL "com.github.actions.icon"="folder-plus"

.

.
</code></pre>
<p>but still they appears in the Github marketplace as an Action. How it’s happening.</p>

**Comment by wipe2238** [comment, marked as answer]
<p><code>action.yml</code> requirement wasn’t enforced during beta until <a href="https://github.blog/changelog/2019-10-10-the-github-actions-marketplace-now-requires-an-actions-metadata-file/" rel="nofollow noopener">this change</a>.</p>

**Comment by ajilraju** [comment]
<p>Thank you for your feedback. :slight_smile:</p>
